examples/aws_iot_shadows.py

```diff
@@ -141,6 +141,7 @@ def message(client, topic, msg):
 client = MQTT.MQTT(
     broker=secrets["broker"],
     client_id=secrets["client_id"],
+    is_ssl=True,
     socket_pool=pool,
     ssl_context=ssl_context,
 )
```

Comment:
@jersu11 can you make sure all 3 look the same way? The native one also as port in it (which really isn't needed). The closer these are, the easier for users just to swap it out for what they need
```py
client = MQTT.MQTT(
    broker=secrets["broker"],
    client_id=secrets["client_id"],
    is_ssl=True,
    socket_pool=pool,
    ssl_context=ssl_context,
)
```

Reply:
Good call out. I've updated the native example to match the MQTT client settings of the other examples.